runtime/j9vm/jvm.c

```diff
@@ -450,6 +450,48 @@ static const J9SignalMapping signalMap[] = {
 #if defined(SIGPOLL)
 	J9_SIGNAL_MAP_ENTRY("POLL", SIGPOLL),
 #endif /* defined(SIGPOLL) */
+#if defined(SIGPWR)
```

Comment:
Is this supposed to be 
```suggestion
#if defined(SIGCLD)
```
?